docs/organization/taxonomy-transform/examples/example-transform.js

```diff
@@ -0,0 +1,57 @@
+module.exports = async function ({ csv, taxonomy, fs }) {
```

Comment:
is this transform function works for example-data?

Reply:
It should, yes 😅  it's from the example script & data you shared

Reply:
It was this one, https://github.com/contentful/contentful-cli/blob/7cc0633ee32e22d51495c1aad9b179efcfa1bbd6/docs/organization/taxonomy-transform/examples/example-spreadsheet.csv it looks different.

Reply:
ah yes, I have changed the text to not copy directly from the customer's provided example